shopfloor_packing_info/views/shopfloor_packing_info_views.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="shopfloor_packing_info_tree_view" model="ir.ui.view">
+        <field name="name">shopfloor.packing.info tree</field>
+        <field name="model">shopfloor.packing.info</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+                <field name="text" />
+                <field name="active" />
+            </tree>
+        </field>
+    </record>
+    <record id="shopfloor_packing_info_form_view" model="ir.ui.view">
+        <field name="name">shopfloor.packing.info form</field>
+        <field name="model">shopfloor.packing.info</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <field name="active" invisible="1" />
+                    <button
+                        type="object"
+                        name="toggle_active"
+                        string="Archive"
+                        attrs="{'invisible': [('active', '=', False)]}"
+                    />
+                    <button
+                        type="object"
+                        name="toggle_active"
+                        string="Restore"
+                        attrs="{'invisible': [('active', '=', True)]}"
+                    />
```

Comment:
Since 13.0 the archived button should use this:

```xml
                    <widget
                        name="web_ribbon"
                        title="Archived"
                        bg_color="bg-danger"
                        attrs="{'invisible': [('active', '=', True)]}"
                    />
```